dev/io.openliberty.jakarta/servlet.6.1.bnd

```diff
@@ -0,0 +1,33 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-SymbolicName: io.openliberty.jakarta.servlet.6.1; singleton:=true
+Bundle-Description: Jakarta Servlet, version 6.1
+
+Export-Package: \
+   	jakarta.servlet;thread-context=true;uses:="jakarta.servlet.annotation,jakarta.servlet.descriptor";version="6.1.0",\
+   	jakarta.servlet.annotation;thread-context=true;uses:="jakarta.servlet";version="6.1.0",\
+   	jakarta.servlet.descriptor;thread-context=true;version="6.1.0",\
+   	jakarta.servlet.http;thread-context=true;uses:="jakarta.servlet";version="6.1.0",\
+   	jakarta.servlet.resources;thread-context=true;version="6.1.0"
+
+-includeresource: \
+  @${repo;jakarta.servlet:jakarta.servlet-api;6.1.0;EXACT}!/!(META-INF/maven/*|module-info.class)
```

Comment:
Not sure how this is working without the update to `cnf/oss_dependencies.maven` update.